desktop/components/my_active_bids/index.styl

```diff
@@ -37,6 +37,10 @@ item-margin-bottom = 16px
 .my-active-bids-item-details
   line-height 20px
 
+.my-active-bids-item-details--hide-bid
+  @extends .my-active-bids-item-details
+  padding-top 10px
+
```

Comment:
Also, if you look at the template it _is_ hiding things, as this class exists within a condition. Post pre-bidding, a handful of sale details are stripped from the view.